frontend-react/src/components/ReportStreamHeader.tsx

```diff
@@ -1,54 +1,162 @@
-import { Header, Title, PrimaryNav, Button, Link } from '@trussworks/react-uswds'
-import { useOktaAuth } from '@okta/okta-react';
-//import { UserClaims } from '@okta/okta-auth-js';
-//import OktaAuthBrowser from '@okta/okta-auth-js/lib/browser/browser';
-import { useEffect, useState } from 'react';
+import {
+    Header,
+    Title,
+    PrimaryNav,
+    Button,
+    Link,
+    Dropdown,
+    NavDropDownButton,
+    Menu,
+} from "@trussworks/react-uswds";
+import { useOktaAuth } from "@okta/okta-react";
+import { useEffect, useState } from "react";
+import { useResource } from "rest-hooks";
+import OrganizationResource from "../resources/OrganizationResource";
 
+/**
+ *
+ * @returns OrganizationDropDown
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const OrganizationDropDown = () => {
+    let orgs = useResource(OrganizationResource.list(), {
+        sortBy: undefined,
+    }).sort((a, b) => a.description.localeCompare(b.description));
 
-const SignInOrUser = () => {
+    let setValue = (e: any) => { };
 
-  const { oktaAuth, authState } = useOktaAuth();
+    return (
+        <Dropdown
+            id="input-dropdown"
+            name="input-dropdown"
+            defaultValue="pima-az-phd"
+            onChange={(e) => setValue(e.target.value)}
+        >
+            {orgs.map((org) => (
+                <option key={org.name} value={org.name}>
+                    {" "}
+                    {org.description} ({org.name})
+                </option>
+            ))}
+        </Dropdown>
+    );
+};
 
-  const [ user, setUser ] = useState('');
+/**
+ *
+ * @returns SignInOrUser
+ */
+const SignInOrUser = () => {
+    const { oktaAuth, authState } = useOktaAuth();
 
-  useEffect( () => {
-    if( authState && authState.isAuthenticated )
-      oktaAuth.getUser().then( cl => setUser( cl.email? cl.email : 'unknown user' ) )
-  });
+    const [user, setUser] = useState("");
 
-  return (    
-    authState && authState.isAuthenticated? 
-    <div className="prime-user-account"><span id="emailUser">{ user? user : '' }</span>
-      <br />
-      <a href="/" id="logout" onClick={()=>oktaAuth.signOut()} className="usa-link">Logout</a>
-    </div> :
-    <Button type="button" outline><Link href="/daily">Sign In</Link></Button>
-  );
-}
+    useEffect(() => {
+        if (authState && authState.isAuthenticated) {
+            oktaAuth
+                .getUser()
+                .then((cl) => setUser(cl.email ? cl.email : "unknown user"));
+        }
+    });
 
-export const ReportStreamHeader = () => {
+    return authState && authState.isAuthenticated ? (
+        <div className="prime-user-account">
+            <span id="emailUser">{user ? user : ""}</span>
+            <br />
+            {authState.accessToken?.claims.organization.some(
+                (o: String) => o === "DHPrimeAdmins"
+            )
+                ? "" //<OrganizationDropDown></OrganizationDropDown>
+                : ""}
```

Comment:
lines 69 and 70 return the same thing. is this intentional?

Reply:
Yeah - the OrgDropDown stuff doesnt work yet - so I just commented it out

Reply:
Yeah - but I see its tripping the Sonar check - so I'll remove it